app/src/main/java/io/token/sample/Application.java

```diff
@@ -104,7 +103,19 @@ private static TokenIO initializeSDK() throws IOException {
      * @return Logged-in member.
      */
     private static Member loginMember(TokenIO tokenIO, String memberId) {
-        return tokenIO.login(memberId);
+        try {
+            return tokenIO.login(memberId);
+        } catch (StatusRuntimeException sre) {
+            if (sre.getStatus().getCode() == NOT_FOUND) {
+                // We think we have a member's ID and keys, but we can't log in.
+                // In the sandbox testing environment, this can happen:
+                // Sometimes, the member service erases the test members.
+                throw new RuntimeException(
```

Comment:
I wanted to catch the case where Token wiped the member db to tell the dev to rm their ./keys dir and start over.

But if there's some other kind of failure, maybe telling them to rm their ./keys dir is bad advice.